script/scaffold/__main__.py

```diff
@@ -72,20 +72,20 @@ def main():
     if args.template != "integration":
         generate.generate(args.template, info)
 
-    pipe_null = "" if args.develop else "> /dev/null"
+    pipe_null = {} if args.develop else {"stdout": subprocess.DEVNULL}
 
     print("Running hassfest to pick up new information.")
-    subprocess.run(f"python -m script.hassfest {pipe_null}", shell=True)
+    subprocess.run(["python", "-m", "script.hassfest"], **pipe_null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4852890)